libs/server/repository/infrastructure/src/lib/bitbucket/repositories/bitbucket-repository.repository.ts

```diff
@@ -34,7 +34,12 @@ export class BitbucketRepositoryRepository extends RemoteRepositoryRepository {
         headers: { Authorization: `Bearer ${token}` }
       })
       .pipe(
-        map((res: AxiosResponse) => res.data),
+        map((res: AxiosResponse) => {
+          if (!(res.data.username || res.data.nickname)) {
+            throw { response: { status: 422 } };
```

Comment:
you can just throw the CoreUnprocessableEntityException form this point with some message like "Cannot resolve the owner of the ${fullName}"

Reply:
But then we will get an error in catchRequestExceptions due to lack of object property response.

Reply:
so maybe we should only process an object with instanceof AxiosResponse, or another discriminator there,